source/basic.tex

```diff
@@ -6935,9 +7004,10 @@
 \pnum
 \indextext{termination!program}%
 \indextext{\idxcode{main} function!return from}%
-A \keyword{return} statement\iref{stmt.return} in \tcode{main} has the effect of leaving the main
-function (destroying any objects with automatic storage duration) and
-calling \tcode{std::exit} with the return value as the argument.
+A \keyword{return} statement\iref{stmt.return} in \tcode{main} has the effect of leaving the main  %JMB: Why is the second main here not in code font?
```

Comment:
Pre-existing bug in ancient wording.  Feel free to drive-by-fix.

Reply:
done